finality-aleph/src/lib.rs

```diff
@@ -194,19 +195,5 @@ where
     C: ClientForAleph<B, BE> + Send + Sync + 'static,
     SC: SelectChain<B> + 'static,
 {
-    let AlephConfig {
-        network,
-        consensus_config,
-        client,
-        select_chain,
-        spawn_handle,
-        auth_keystore,
-        authorities,
-    } = config;
-    let consensus = party::ConsensusParty::new(client, network, select_chain, auth_keystore);
-    async move {
-        consensus
-            .run(authorities, consensus_config, spawn_handle.into())
-            .await;
-    }
+    run_consensus_party(AlephParams{ config })
```

Comment:
Earlier, the `run_aleph_consensus` permuted the fields of `AlephConfig`, put some of them to the `ConsensusParty` and passed remaining ones in the `run` method. Instead we can simply pass the config to an async function.  The config is wrapped in an `AlephParams` type, mimicking the style of grandpa.